aclk/agent_cloud_link.c

```diff
@@ -1128,8 +1135,15 @@ int aclk_init(ACLK_INIT_ACTION action)
     aclk_hostname = config_get(CONFIG_SECTION_ACLK, "agent cloud link hostname", ACLK_DEFAULT_HOST);
     aclk_port = config_get_number(CONFIG_SECTION_ACLK, "agent cloud link port", ACLK_DEFAULT_PORT);
 
+    aclk_proxy = config_get(CONFIG_SECTION_ACLK, ACLK_PROXY_CONFIG_VAR_NAME, "");
```

Comment:
There are very few places in the config that use empty strings as defaults, the common pattern is to use a well-defined constant to turn it off such as "no" or "none" or "disabled".